locals.tf

```diff
@@ -245,7 +245,7 @@ locals {
     qa          = 150
     integration = 150
     preprod     = 170
-    production  = 170
+    production  = 269
```

Comment:
did we do the maths to make sure this many will fit?

Reply:
yep (number of executors per instance * number of core instances) minus 1 for the driver